data/jobs.txt

```diff
@@ -2197,6 +2197,26 @@ mission "Colonists [1]"
 		payment 10000
 		dialog "The colonists depart your ship after paying you <payment>."
 
+mission "Stranded field trip to <planet>"
+	job
+	repeat
+	description `These <bunks> school students have been stranded on an educational field trip due to the unexpected bankruptcy of their booked transport. They have the supplies to continue their studies but need to be safely returned home at some point. The school & parents have put up a reward of <payment>.`
+	passengers 20 100 .6
+	cargo "educational supplies" 2 10 .6
+	to offer
+		random < 10
+		"passenger space" > 50
+	source
+		attributes "near earth" "deep" "paradise" "tourism"
```

Comment:
Did you mean for all sources to have the "tourism" attribute, and all destinations to have the "urban" attribute? If so, you need a second `attributes` node for each filter, with the respective required attribute on that line.

Reply:
No the intent here was that these jobs would mostly be available within the developed part of human space, but that occasionally there'd be some that were found on more distant tourist style locations that need to get back, or some in the more developed part that had come from a more distant urban world and needed returning.